server/src/main.rs

```diff
@@ -204,21 +187,25 @@ mod tests {
         assert_test_server_responds_with!(
             "/repo/idontexist?file=README.md&reference=origin/master",
             404,
-            ""
+            "No repo found with name 'idontexist'"
```

Comment:
We now get error responses back too, finally — you can see there's a few other tests below where I now assert error responses (not just status) now. 🎉